frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/ParametersSettings.tsx

```diff
@@ -62,10 +62,13 @@ export const ParametersSettings = ({
       getValuePopulatedParameters(
         lockedParameters,
         parameterValues,
+        /* defaultRequired */ true,
       ) as EmbedResourceParameterWithValue[],
     [lockedParameters, parameterValues],
   );
 
+  const hasRequiredParameters = resourceParameters.some(p => p.required);
```

Comment:
If possible try to use the unabbreviated names to ensure readers don't have to think twice what each variable would represent. I understand it won't make much difference in this case, but I've seen some folks get used to doing this all the time and we could see a bunch of abbreviated names in the function which could be confusing.